ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,381 @@
+                                        ወኢተፍጻሜት፡<hi rend="rubric"> እምላክነ፡ በመለኮቱ፡ ወአቡነ፡ በኂሩቱ።</hi>
+                                    ዘስብሐቲሁ፡ እምኀቤሁ፡ ወውዳሴሁ፡ እምዚአሁ፡ ዘይክል፡ ኵሎ፡ ወአልቦ፡ ዘይስአኖ፡ ለዓለመ፡ ዓለም፡
+                                    አሜን፨ አጽሕፍ፡ እንከ፡ ንስቲተ፡ አነ፡ ኃጥእ፡ ገብራ፡ <space unit="chars"
+                                        quantity="7"> </space> እምዜና፡ ስደራ፡ ወእ<hi rend="rubric">ምብዙኅ፡
+                                        ገድላቲሃ፡ ወትሩፋቲሃ፡ ለእምነ፡ ቅድስት፡ ወለተ፡ ጴጥ<cb/>ሮስ፡ ሰጻፍ፡ እመ፡ አዕናቊ፡
+                                        አዕላፍ ። ። ።</hi>
+                                </incipit>
+                                <note>According to <bibl><ptr
+                                            target="bm:WendyundKleiner2015WalattaPetros"
+                                            /><citedRange>271-272</citedRange></bibl>, the present
+                                    manuscript has an <cit>
+                                        <quote>alternate ending</quote>
+                                    </cit> in comparison to other witnesses.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.2.">
+                                <locus from="147va" to="171vb"/>
+                                <title type="complete" ref="LIT7073TammeraW">Taʾammǝra Walatta
+                                    Ṗeṭros</title>
+                                <incipit xml:lang="gez">
+                                    <locus from="147va" to="148rb"/>
+                                    <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ </hi>፩አምላክ፡ ዘለዓለም።
+                                    ዓዲ፡ ናተሉ፡ ጽሒፈ፡ <hi rend="rubric">ተአምረ፡ ዘገብረት፡ እምነ፡ ቡርክት፡ ወለተ፡
+                                        ጴጥሮስ። ጸሎታ፡ ወ</hi>በረከተ፡ ረድኤታ፡ የሀሉ፡ ምስሌነ፡ ለዓለመ፡ ዓለም፡ አሜን። ስምዑ፡
+                                    ንንግርክሙ፡ ኃይለ፡ ተአምር፡ ዘኮነ፡ እምድኅረ፡ ዕረፍታ፡ ለእምነ፡ ብፅዕት፡ <hi
+                                        rend="rubric">ወለተ፡ ጴጥሮስ፡</hi> ዘሰማዕነ፡ ወዘርኢነ፡ ዘይትገበር፡ በዕለተ፡
+                                    ተዝካራ፡ ለለጊዜሁ፡ እም፴ዕለት፡ እስከ፡ ፍጻሜ፡ ዓመት፡ ከመ፡ ወደረ፡ በረከተ፡ እግዚአብሔር፡ ላዕለ፡
+                                    ኵሉ፡ ዘኮነ፡ ግብረ፡ ተዝካር፡ ዘይትገበር። ወአሜሃ፡ ተጋብኡ፡ ብዙኃን፡ ሰብእ፡ እምሥራቅ፡
+                                    ወእምዕራብ፡ እምአድባር፡ ወእመካናት፡ ወእምገዳማት፡ ወእምኵሉ፡ አኅጉር፡ እለ፡ አልቦሙ፡ ኍልቊ።
+                                    ቦእምኔሆሙ፡ መምህራን፡ ወቀሳውስት፡ ወዲያቆናት፡ ወመነኮሳት፡ ምስለ፡ ማዕጠንታት፡ ወመስቀላት። ወቦ፡
+                                    እምኔሆሙ፡ ነዳያን፡ ወምስኪናን፡ ወዕውራን፡ ወሐንካሳን፡ እስመ፡ ተሰምዓ፡ ዜናሃ፡ ውስተ፡ ኵሉ፡
+                                    አጽናፈ፡ ዓለም። ወወሀብዎሙ፡ ብዙኃ፡ መብልዐ፡ ወመስቴ፡ በበማዕርጊሆሙ፡ ወበበመጠሆሙ፡ እስከ፡ ጸግቡ፡
+                                    ወአትረፉ። ፩ተአምር፡ ዘኮነ፡ ሶቤሃ። ወይቤሉ፡ ሀለወ፡ ኅዳጥ፡ ኅቡአ፡ መጠነ፡ ፩ኅብስት፡ በዓቢይ፡
+                                    ጋን፡ ዘተረፈ፡ እምነ፡ ተዝካር። ወእምዝ፡ ተረክበ፡ እንዘ፡ ይፈልሕ፡ ወይመልዕ፡ እስከ፡ ላዕሉ፡
+                                    ወይትከዓው። </incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.3">
```

Comment:
i think this is an explicit of type supplication, not an msItem
https://github.com/BetaMasaheft/Documentation/issues/946
https://github.com/BetaMasaheft/Documentation/issues/1003

if it is a regular part of the work then a div should be created for it under work
https://betamasaheft.eu/Guidelines/?id=ColophSupplTit

Reply:
Thank you for the links! I have changed the encoding and added a note, explaining that the supplication and the colophon are not a regular part of the work.